content/docs/setup/kubernetes/spec-requirements/index.md

```diff
@@ -38,3 +38,70 @@ cluster must satisfy the following requirements:
 
 * _**`NET_ADMIN` capability**:_ If pod security policies are enforced in your cluster and unless you use the [Istio CNI Plugin](/docs/setup/kubernetes/istio-cni-install/), your pods must have the `NET_ADMIN` capability allowed.
 See [Required Pod Capabilities](/help/ops/setup/required-pod-capabilities/).
+
+* _**Check required ports**_:
```

Comment:
How is this an application pod/service requirement? Check ports how?

Reply:
Added some text on verifying ports. Basically, one has to verify that the Istio ports are available before installing Istio or there will be a port conflict. Similarly, their services may not start post install if they are using a port that Istio is already using. These would be requirements on their services so why I thought to put on this page. I'm fine moving it to a better spot.

Reply:
First of all, I don't think it makes any sense to embed this big table in the bulleted list of pod/service requirements. If you want to put the table in the same document, at least move it to a separate section, for example:
```
## Istio Required Ports

Make sure the following ports are available before installing Istio components.

<table> ...
```

Then, you can have a entry in the pod/service requirements list that says something like:
```
***Pod and Service Ports***: Ensure that your ports are available and don't conflict with the [Istio required ports](#istio_required_ports).
```
That said, I'm not sure exactly what the actual requirement is. Is it pod ports can't conflict, service ports can't conflict, both? HTTP ports, TLS ports, both? Does the NodePort also matter? You need to document the restriction accurately.

Reply:
I did put the table in a different section and referenced from the bulled list.

I also restructured the table to list by port and include the list of components using the port/protocol.

Greg also helped with creating some text to describe the issues between protocol usage for each port.